tests/clients/test_experiment.py

```diff
@@ -662,7 +699,8 @@ def test_experiment_client_list_runs_page(mocker):
     )
 
 
-def test_log_run_data(mocker):
+@pytest.mark.parametrize("metric", [METRIC, METRIC_WITHOUT_STEP])
```

Comment:
Is this necessary? We are just testing that `ExperimentRunDataSchema` has been called, not that it has done anything with the input, so `METRIC` and `METRIC_WITHOUT_STEP` are functionally equal in this test.